ghost/portal/src/components/TriggerButton.styles.js

```diff
@@ -27,7 +27,7 @@ const TriggerButtonStyles = `
         background: var(--brandcolor);
         height: 60px;
         min-width: 60px;
-        box-shadow: rgba(0, 0, 0, 0.24) 0px 8px 16px -2px;
+        box-shadow: rgba(var(--blackrgb), 0.24) 0px 8px 16px -2px;
```

Comment:
The trigger button shadows and outlines are chosen to work in most situations regardless of the background color. They are not supposed to be changed, let's keep them hardcoded.